CHANGELOG.rst

```diff
@@ -1,5 +1,12 @@
 Changelog
 =========
+1.20.1 (2021-08-26)
```

Comment:
Could you please pull from the master branch to remove these unintended changes? Thanks.

Reply:
ok, done.